src/algorithms/tracking/TrackParamTruthInit.cc

```diff
@@ -73,10 +73,7 @@ eicrecon::TrackParameters *eicrecon::TrackParamTruthInit::produce(const edm4hep:
     }
 
     // modify initial momentum to avoid bleeding truth to results when fit fails
-    // this picks uniformly between [1-eps,1,1+eps] times true momentum, then smeared
-    const auto pinit = pmag
-                     * (1.0 + m_cfg.m_momentumSplit * m_uniformIntDist(generator))
-                     * (1.0 + m_cfg.m_momentumSmear * m_normDist(generator));
+    const auto pinit = pmag*(1.0 + m_cfg.m_momentumSmear * m_normDist(generator));
```

Comment:
This PR should also remove the `m_momentumSplit` parameter since it is now unused.

Reply:
(from the configuration, that is)

Reply:
Yes @wdconinc you are right. Sorry, I selected the option as a bug but  I changed  it to an other option. It is kind of small changes we want it as discussed on email.

Reply:
Any progress on removing the `m_momentumSplit` parameter?

Reply:
Thanks, Yes I removed it from the configurable and also from the global configurable. I also changed default momentum smearing to 10%.